testcases/testcase1/models/tags.json

```diff
@@ -0,0 +1,22 @@
+{
+  "oveAct_u": {
+    "writable": "m:",
+    "unit": "s:W"
+  },
+  "PHea_y": {
+    "point": "m:",
+    "sensor": "m:",
+    "unit": "s:W"
+  },
+  "TRooAir_y": {
+    "point": "m:",
+    "sensor": "m:",
+    "temp": "m:",
+    "unit": "s:C"
```

Comment:
This unit should be K.

Reply:
> @kbenne I think eventually it would be good to at least be able to specify (in a test case) and get (as a user) a full haystack model, which if I understand correctly could be implemented as a json of tags. Does it require a full haystack API implementation to get the haystack model, or does one without the other sound ridiculous? A haystack API could indeed live alongside and be complementary to the current BOPTEST API, but with notable overlaps in functionality being `PointWrite`-`/advance` (though advance also advances the simulation) and `HisRead`-`/results`.
> 
> In the meantime for this PR, can you :
> 
> * Address my inline comments and
> * Update design documentation to include `tags.json` in the [test case directory spec](https://github.com/ibpsa/project1-boptest/blob/master/docs/DesReqGui/source/testcasedev.rst#test-case-repository) (comment that it is optional right now), and also add a section and spec for `tags.json` as you've implemented similar to [other json specs](https://github.com/ibpsa/project1-boptest/blob/master/docs/DesReqGui/source/testcasedev.rst#time-period-specification-and-json-mapping).
> * Update [release notes](https://github.com/ibpsa/project1-boptest/blob/master/releasenotes.md) with this edit

As far as I know, you don't have to implement all of the Haystack API. In fact the API specifies an `/ops` endpoint that returns a form of documentation for what endpoints the server is supporting. I think `read` and `nav` would be sufficient for model discovery.

Reply:
Then the use of ``/ops``, ``read``, and ``nav``, along with implementing an example "full" model, I think could be a good next goal, at least in the direction of haystack support.

Reply:
For https://github.com/ibpsa/project1-boptest/pull/361#discussion_r718007783 the unit has been changed in https://github.com/ibpsa/project1-boptest/pull/361/commits/0868a08aab0dc14fd93ddd9d26c21c773dd2ab9c.